synapse/appservice/__init__.py

```diff
@@ -251,6 +264,15 @@ def get_exlusive_user_regexes(self):
             if regex_obj["exclusive"]
         ]
 
+    def get_groups_for_user(self, user_id):
+        """Get the groups that this user is associated with by this AS
+        """
```

Comment:
Document types please:

```
Args:
    user_id (str)

Returns:
    iterable[str]: returns an iterable that yields group_id strings
```

or something

Reply:
I forgot to do this